app/gateway-manager/control-plane-groups.md

```diff
@@ -107,6 +125,40 @@ flowchart LR
 * The Control Plane Group also contains Control Plane Purple, which is managed by a central platform team.
 * The central platform team manages global plugin configuration in Control Plane Purple, which is added to any configuration that teams Blue and Green provide.
 
+## How do I create a Control Plane Group?
+
+In {{site.konnect_short_name}} Control Plane Groups can be created using the [Control Planes API](/api/konnect/control-planes/).
+<!--vale off-->
+{% control_plane_request %}
+method: POST
+url: /v2/control-planes
+status_code: 201
+headers:
+  - 'Authorization: Bearer $KONNECT_TOKEN'
+  - 'Content-Type: application/json'
+body:
+  name: CPG
+  cluster_type: CLUSTER_TYPE_COMPOSITE
```

Comment:
FYI, the API spec doesn't include an entry for `CLUSTER_TYPE_COMPOSITE`. Guessing that needs an update (non-blocking for this PR, if that value exists)

https://kongdeveloper.netlify.app/api/konnect/control-planes/v2/#/operations/create-control-plane